app/contexts/surveys/action_context.rb

```diff
@@ -3,23 +3,28 @@ class ActionContext
 
     private
 
-    attr_reader :survey, :group
+    attr_reader :group, :survey
 
     public
 
+    attr_reader :form
+
     delegate :logo_url,       to: :group, prefix: true
     delegate :manager_email,  to: :group, prefix: true
     delegate :name,           to: :group, prefix: true
+    delegate :slug,           to: :group, prefix: true
     delegate :website_url,    to: :group, prefix: true
 
     delegate :when_group_has_website_url, to: :group_presenter
+    delegate :when_manager,               to: :group_presenter
```

Comment:
Yes, 2 times in [groups/information_header](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/_information_header.html.slim) partial that is called from [surveys/new](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/surveys/new.html.slim) and [surveys/edit](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/surveys/edit.html.slim)